### DISCUSSION TITLE
error on FES2014 data Tidal Correction

### DISCUSSION THREAD

**Discussion by luca-sobreira** [kvos/CoastSat#493] [Q&A] [answered]
Hi, @kvos, here we are trying to aplicate CoastSat on our area. Our tide data wasn´t good, it doesn´t couver the temporal series. We were  trying to use the FES2014 with CoastSat Slope to get the data. We get they on a .csv file using pandas lib. An error appered on this step, showing that the data is not valid because it is not every 15 minutes or every hour. how can we solve this using FES2014 data? thanks for this very useful tool :) 

the error message and our file generated with pandas lib
[teste.csv](https://github.com/kvos/CoastSat/files/15151452/teste.csv)
![Captura de tela 2024-04-29 122857](https://github.com/kvos/CoastSat/assets/149187844/4eb1a9d7-a243-4535-8f95-a3e8fe5ae305)

**Comment by kvos** [comment]
ok, can you print dates_sat, dates_ts and tides_ts, then I'll be able to tell what's causing the error.

**Comment by luca-sobreira** [comment]
![Captura de tela 2024-04-30 100646](https://github.com/kvos/CoastSat/assets/149187844/132cd8c5-3949-43d5-8bac-ce22e690282b)
![Captura de tela 2024-04-30 100658](https://github.com/kvos/CoastSat/assets/149187844/34c74479-1f8e-4785-b44f-64d2b8e1da55)
![Captura de tela 2024-04-30 100711](https://github.com/kvos/CoastSat/assets/149187844/e5f47f26-b5d3-4793-8af2-daee61ac27c3)
@kvos

**Reply by kvos** [reply, marked as answer]
your problem is with `dates_ts`, it hasn't been able to read the dates from the csv correctly as it filling in with NaT, try to adjust the parameters of pd.read_csv so that it reads the timestamps properly

**Reply by luca-sobreira** [reply]
thanks for the answer, we will try to adjust this parameters
